covid-vaccine-faq-bot/assets/index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="x-ua-compatible" content="ie=edge">
+        <title>Get started with your Twilio Functions!</title>
+        <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Roboto:300,300italic,700,700italic">
+        <link rel="stylesheet" href="https://unpkg.com/normalize.css/normalize.css">
+        <link rel="stylesheet" href="https://unpkg.com/milligram/dist/milligram.min.css">
+        <link rel="stylesheet" href="/styles.css">
+    </head>
+    <body>
+        <header>
+            <h1>Welcome!</h1>
+            <h1>COVID-19 Vaccine FAQ SMS Bot</h1>
```

Comment:
Page looks a little funny with two `<h1>`s, can we remove the first one?

Reply:
done